docs/source/scripts.rst

```diff
@@ -0,0 +1,56 @@
+..  scripts:
+
+Utility scripts
+===============
+
+Visualize data augmentation transformations
+***********************************************
+Data augmentation is a key part of the Deep Learning training scheme. This script aims at facilitating the fine-tuning of data augmentation parameters. To do so, this script provides a step-by-step visualization of the transformations that are applied on data.
+
+The ``scripts/visualize_transforms.py`` script applies a series of transformations (defined in a configuration file ``-c``) to ``-n`` 2D slices randomly extracted from an input image (``-i``), and save as png the resulting sample after each transform.
+
+For example::
+
+    python scripts/visualize_transforms.py -i t2s.nii.gz -n 1 -c config.json -r t2s_seg.nii.gz
+
+Provides a visualisation of a series of three transformation on a randomly selected slice:
+
+.. image:: ../../images/transforms_im.png
+    :width: 200px
+    :align: center
+    :height: 100px
+
+And on a binary mask::
+
+    python scripts/visualize_transforms.py -i t2s_gmseg.nii.gz -n 1 -c config.json -r t2s_seg.nii.gz
+
+Gives:
+
+.. image:: ../../images/transforms_gt.png
+    :width: 200px
+    :align: center
+    :height: 100px
+
+Convert a model from PyTorch to ONNX format
```

Comment:
same here about length, how about "Convert to ONNX"
but the more i think about it, the more i like the "verbose" approach. Not sure how it will appear on the TOC. we could ask RTD to build the doc for this branch. i'll do it now...